### PR TITLE
Kat10140/homepage demos

### PR DESCRIPTION
**Pull request by kat10140**
**Related Issue:** #134 #137 

added:
- blurb on homepage
- links to readmes/sourcecode in the demos
- links to demos in the readmes

### CODE REVIEW

File: src/demos/calcite-action-bar.html [hunk 1]
```diff
@@ -32,7 +33,7 @@ <h1>calcite-action-bar</h1>
             height: 100%;
           }
         </style>
-        <div class="action-bar-container">
+        <div class="action-bar-container">
```
Comment (driskull) [inline src/demos/calcite-action-bar.html]: Some rogue spaces here

File: src/components/calcite-action-bar/readme.md [hunk 2]
```diff
@@ -4,8 +4,9 @@
 
 The `calcite-action-bar` component is made up of multiple `calcite-actions` in the form of clickable icons. The action bar can be expanded to view actions with descriptive text or made smaller to view with just icons.
 
-<!-- Auto Generated Below -->
+See the demo [here](https://esri.github.io/calcite-app-components/demos/calcite-action-bar.html).
```
Comment (driskull) [inline src/components/calcite-action-bar/readme.md]: Can we link the word “demo” instead of “here”? This applies other places in the PR.

Comment (kat10140) [reply src/components/calcite-action-bar/readme.md]: do you want me to change the link in the readmes as well? they follow the same pattern

Comment (kat10140) [reply src/components/calcite-action-bar/readme.md]: i mean the link TO the readmes in the demos

Comment (driskull) [reply src/components/calcite-action-bar/readme.md]: Yeah 👍 

For accessibility and readability, text should describe the link location so `See the [action-bar demo]` would be better.

**Comment by kat10140** [comment]
@driskull done :)

**Pull request merged**